lib/puppet/type/protocol_properties.rb

```diff
@@ -24,7 +24,7 @@
         end
     end
 
-    newparam(:protocol) do
+    newparam(:name) do
```

Comment:
Renamed. All known examples are setting name via the name/title of the resource.